### DISCUSSION TITLE
PO-MODAL - É possivel exibir um pdf?

### DISCUSSION THREAD

**Discussion by emnovaes** [po-ui/po-angular#821] [Tire suas dúvidas]
Olá!

É possível abrir um **po-modal** exibindo um PDF para leitura (pdf-viewer)?
Caso não seja o componente indicado, teria um outro que apresente este comportamento?

Grato e parabéns pelo projeto.

**Comment by jhosefmarks** [comment]
Você consegue colocar o que quiser dentro do modal, é só adicionar o elemento no corpo do componente.

Atualmente não tem nenhum componente específico para exibição de PDF no PO UI, você vai precisar adicionar na mão mesmo.
